### ISSUE TITLE
CA1822 Mark Members As Static should not trigger for NUnit test methods

### ISSUE THREAD

**Issue by n-sidorov** [dotnet/roslyn-analyzers#1693]
#### Analyzer package

Microsoft.CodeQuality.Analyzers, version [2.6.0](https://www.nuget.org/packages/Microsoft.CodeQuality.Analyzers/2.6.0)

#### Analyzer

MarkMembersAsStatic (CA1822)

#### Repro steps

Any NUnit test method that doesn't use test fixture members.

Example:
```csharp
using NUnit.Framework;

namespace TestStaticAnalyzer
{
    [TestFixture]
    public class TestClass
    {
        [Test]
        public void TestMethod()
        {
            Assert.Fail();
        }
    }
}
```

Tested with `.Net Core 2.0` (project targeting `netcoreapp2.0`) and `NUnit 3.10.1`.

#### Expected behavior
Rule should not be triggered. 
Same class with MSTest attributes does not trigger this rule.

#### Actual behavior
Rule triggered
` warning CA1822: Member TestMethod does not access instance data and can be marked as static (Shared in VisualBasic)`

**Comment by 333fred** [comment]
Appears to be a duplicate of https://github.com/dotnet/roslyn-analyzers/issues/1399, which was fixed just after 2.6.0 was released. @mavasani, do we know when we'll be releasing new packages yet?

**Comment by mavasani** [comment]
> do we know when we'll be releasing new packages yet?

Waiting for @DustinCampbell to merge in his MSBuild workspace analyzer.

**Comment by DustinCampbell** [comment]
Sorry, I'm working on the docs now. Maybe I should create an ask.ms link to use in the analyzer's message?
